### ISSUE TITLE
Changes to End Gcode not saved

### ISSUE THREAD

**Issue by smartavionics** [Ultimaker/Cura#2128]
I'm using the master branch and I just tried adding to the gcode in the End Gcode text box. I close the dialog and when I reopen it, the changes have disappeared. Other fields in that dialog appear to save changes OK.

Ahh, just noticed I get these errors:

```
file:///usr/local/share/cura/resources/qml/Preferences/MachinesPage.qml:120: TypeError: Cannot read property 'onCompleted' of null

file:///usr/local/lib/cura/plugins/MachineSettingsAction/MachineSettingsAction.qml:758: ReferenceError: gcodeField is not defined
```

**Comment by smartavionics** [comment]
Hmm, perhaps gcodeField should be gcodeArea?

````
 propertyProvider.setPropertyValue("value", gcodeField.text)
````

**Comment by smartavionics** [comment]
Ha, it was 4705ada6e, fixed in #2129
